department-of-veterans-affairs/vets-website
# Supplemental Claim | Update introduction page

# Original Ticket

[#48156](https://github.com/department-of-veterans-affairs/va.gov-team/issues/48156)

## URL of change

`/decision-reviews/supplemental-claim/request-supplemental-claim-form-20-0995/`

## Background

Design & content for the Supplemental Claim introduction page has been complete and has been implemented in this PR.
Additionally, the form cannot be started by a Veteran that is not LOA3 verified, so an additional view has been added.

<details><summary>Logged out intro page</summary>


![Logged out view only contains one log in button near the bottom of all the intro page content](https://user-images.githubusercontent.com/136959/198327362-8a32964c-504b-4ccc-a074-0ee51dc8a400.png)</details>

<details><summary>Signed in but unverified, showing alert</summary>


![verify your identity alert shown at the top of the page with a link to the verify page](https://user-images.githubusercontent.com/136959/198327355-8fe07e2a-c9e6-4a40-a59a-378a4ee43415.png)</details>

<details><summary>Signed in and ready to start form</summary>


![start your claim action links shown above and below the process list](https://user-images.githubusercontent.com/136959/198327344-ad6c58cf-f8e1-4881-aa84-d539e97d6b5b.png)</details>

## Steps to test

1. Pull this branch locally, or test in the review instance
2. Go to the URL of change
3. Check the logged out view
4. Log into an unverified account (create a new one in ID.me, but don't verify), then view the intro page
5. Log into user 228 (Colder) and view the intro page with start the form action links

## Code changes

- Updated intro page content to match the [content source of truth](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/decision-reviews/Supplemental-Claims/design/sourceofcontenttruth.md#file-a-supplemental-claim)
- The [design was also maintained](https://www.sketch.com/s/d2416db4-9a4f-4919-abe4-20ba4bdcfd89/p/B3A2055C-DABF-4873-8EB5-846DCFDD0AB3/canvas#Version)
- Changed "Request a Supplemental Claim" to "File a Supplemental Claim" - will need additional work & IA review to finalize URL and breadcrumbs
- Added verify alert which is shown when the user is not LOA3 verified
- Added intro page unit tests
- Updated Cypress e2e tests to bypass the subtask and test the intro page

## How was your code tested

- Added intro page unit tests
- Updated e2e test
- Manual a11y testing with axe-core, HeadingsMap & WAVE

## Acceptance criteria
- [x] Intro page content & design updated
- [x] Add LOA3 requirement to start form 

## Definition of done
